app/ui-react/packages/auto-form/src/widgets/FormArrayComponent.tsx

```diff
@@ -124,6 +124,7 @@ export const FormArrayComponent: React.FunctionComponent<IFormArrayControlProps>
         <div className={'form-array-control__array-add'}>
           <Button
             variant={'link'}
+            className={'pf-m-secondary'}
```

Comment:
I think this actually should be used like `<Button variant={'secondary'}/>` and not by adjusting the classname.